front/src/Components/AuthentificationForm.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
```

Comment:
import everything should not be required and you aren't using anything from this omport